Makefile

```diff
@@ -22,3 +22,6 @@ dev-app: install
 
 test-app: install
 	pipenv run gunicorn --bind=0.0.0.0:5000 'orca:create_app("config.TestConfig")'
+
+clean-tmp:
+	docker exec -it orca rm /tmp/orca-output-*
```

Comment:
Deleting all the tmp files could delete files that are currently in use. You might consider doing something like this: 
`docker exec -it orca find /tmp -name orca-output* -type f -mtime +1`

So you only remove files from 1/2 days ago instead of today. Though switching this to a tmpfs mount would clear on stop so this would only be applicable to a running container.